proof/refine/RISCV64/Invariants_H.thy

```diff
@@ -728,8 +1031,8 @@ where
 | "runnable' (Structures_H.Inactive)                = False"
 | "runnable' (Structures_H.Restart)                 = True"
 | "runnable' (Structures_H.IdleThreadState)         = False"
-| "runnable' (Structures_H.BlockedOnReceive a b)    = False"
-| "runnable' (Structures_H.BlockedOnReply)          = False"
+| "runnable' (Structures_H.BlockedOnReceive _ _ _)  = False"
+| "runnable' (Structures_H.BlockedOnReply _)        = False"
```

Comment:
:+1:

Looks like I didn't clean this one up in my pass back then (probably hadn't touched this part). If you feel like it, you could remove the `Structures_H` and make this a `fun` with default case `False`. This could just be written as `runnable' st == st = Running \/ st = Restart`, but I have the feeling we wanted the `simp` rules we get from primrec/fun